concrete/controllers/single_page/dashboard/extend/install.php

```diff
@@ -168,8 +168,9 @@ public function install_package($package)
 
     public function package_installed($pkgID = 0)
     {
-        $this->set('message', t('The package has been installed.'));
-        $this->set('installedPKG', Package::getByID($pkgID));
+        $pkg = Package::getByID($pkgID);
+        $this->set('message', t('Package "%s" version %s has been installed.', $pkg->getPackageName(), $pkg->getPackageVersion()));
```

Comment:
`t($pkg->getPackageName())` instead of `$pkg->getPackageName()`.

Also, as general rule of thumb, when multiple placeholders (eg `%s`) are present in a translatable string, I'd use the positional syntax.

For example, in this case I'd write:

```php
t('Package "%1$s" version %2$s has been installed.', $name, $version)
```

This allows translators to change the order of the placeholders.

For instance, in Italian this sentence sounds better if we say `Version %2$s of package %1$s has been installed.`)